app.js

```diff
@@ -4,7 +4,8 @@ var express = require('express'),
     compression = require('compression'),
     helmet = require('helmet'),
     frameguard = helmet.frameguard,
-    reactRouted = require('./dist/lib/react-server-route.js');
+    reactRouted = require('./dist/lib/react-server-route.js'),
+    getLocale = require('./dist/lib/get-locale.js');
```

Comment:
getLocale figures out if a locale is supported and a real locale, and returns the best suggested locale given the params.

The params are getLocale(headerLocale, urlPathLocale)

So, if urlPathLocale is not valid, it'll figure out the locale based on the header info passed in.